src/constants.js

```diff
@@ -10,5 +10,19 @@ module.exports = {
     "No courses found!  For more information, see README.md",
   AWS_REGEX: new RegExp(
     /https?:\/\/open-learning-course-data(.*)\.s3\.amazonaws.com/g
-  )
+  ),
+  BOILERPLATE_MARKDOWN: [{
+      path: "",
+      name: "_index.md",
+      content: `---\ntitle: Hugo Course Publisher\n---\n`
+    }, {
+      path: "search",
+      name: "_index.md",
+      content: `---\ntitle: Search\ntype: search\n---\n`
+    }, {
+      path: "courses",
+      name: "_index.md",
+      content: `---\ntitle: Courses\ntype: courseindex\n---\n`
```

Comment:
one thing I might suggest is storing the contents as objects here - like `content: { title: "Hugo Course Publisher" }` and then using `yaml.safeDump` to convert them when we write the files - I'm just thinking that will make it a little easier to edit these values in the future if we need to add a property

Reply:
good idea!